manifests/a/Avogadro/Avogadro/1.2.0/Avogadro.Avogadro.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Avogadro.Avogadro
+PackageVersion: 1.2.0
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://jztkft.dl.sourceforge.net/project/avogadro/avogadro/1.2.0/Avogadro-1.2.0n-win32.exe

```

```suggestion
  InstallerUrl: https://sourceforge.net/projects/avogadro/files/avogadro/1.2.0/Avogadro-1.2.0n-win32.exe/download
```
Just in case the mirror URL goes down.